src/Schema.ts

```diff
@@ -136,12 +139,23 @@ export interface DoubleClickStep extends ClickAttributes, StepWithSelectors {
 export interface ClickStep extends ClickAttributes, StepWithSelectors {
   type: StepType.Click;
   /**
-   * Delay (in ms) between the mouse up and mouse down of the click. Defaults to
-   * 50ms.
+   * Delay (in ms) between the mouse up and mouse down of the click.
+   *
+   * @defaultValue `50`
    */
   duration?: number;
 }
 
+export interface DragStep extends ClickAttributes, StepWithTarget {
```

Comment:
I don't think there is value in a drag step based on coordinates. We should do the following instead https://github.com/puppeteer/replay/issues/273#issuecomment-1223606241

Reply:
Rationale: 1) no one is going to define coordinates by hand so recording part is a must 2) coordinates are fragile and any change in the viewport/website will break the recorded drag.

If it's for the drawing use case, let's use maybe a different name but the point 1 still applies. Without Recording, it is difficult to use. Let's, therefore, re-consider the whole story starting with the use case.

Reply:
I agree. While designing this it didn't make sense to use coordinates for drag 'n drop.

One thing I wanted as a requirement for this feature is to ensure drag n' drop and "draw"ing were the same API. It's not reasonable to separate them since from the user point of view, they are the same motion. Because of this, it's fairly important we find the correct granularity so we can do this.